README.md

```diff
@@ -1,8 +1,92 @@
-⚠️ ⚠️ ⚠️
+# ⚠️ `@github/webauthn-json` is deprecated ⚠️
 
-WebAuthn-json has been sunset. Now that [all major browsers support WebAuthn](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#browser_compatibility) we recommend invoking the native APIs
+As of March 2025, stable versions of all major browsers now support the following methods:
 
-⚠️ ⚠️ ⚠️
+- [`PublicKeyCredential.parseCreationOptionsFromJSON(…)`](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/parseCreationOptionsFromJSON_static)
+- [`PublicKeyCredential.parseRequestOptionsFromJSON(…)`](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential/parseCreationOptionsFromJSON_static)
+- [`PublicKeyCredential` → `.toJSON()`](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential)
+
+These should be used instead of `@github/webauthn-json`.
+
+## 👉 Use built-in browser methods instead 👈
+
+Here is an example for how to serialize and deserialize JSON for WebAuthn client code without using `@github/webauthn-json`:
+
+```ts
+// Example in TypeScript
+
+const jsonWebAuthnSupport = !!globalThis.PublicKeyCredential?.parseCreationOptionsFromJSON;
+
+async function register() {
+  const publicKey = PublicKeyCredential.parseCreationOptionsFromJSON({ /* … */ });
+  const credential = (await navigator.credentials.create({publicKey})) as PublicKeyCredential;
+  return credential.toJSON();
+}
+
+async function authenticate() {
+  const publicKey = PublicKeyCredential.parseRequestOptionsFromJSON({ /* … */ });
+  const credential = (await navigator.credentials.get({publicKey})) as PublicKeyCredential;
+  return credential.toJSON();
+}
+
+if (jsonWebAuthnSupport) {
+  /* Set up code to call `register()` and `authenticate()`. */
+}
+```
+
+## Reasoning
+
+`@github/webauthn-json` served as an ecosystem prototype of the functionality was [developed into the built-in browser methods](https://github.com/w3c/webauthn/wiki/Explainer:-JSON-Serialization-Methods). The built-in methods are compatible with `@github/webauthn-json` encoding, so you can use them as a drop-in substitute. We strongly recommend doing so, since:
```

Comment:
This feels grammatically off to me.

```suggestion
`@github/webauthn-json` served as an ecosystem prototype of the functionality while it was [developed into the built-in browser methods](https://github.com/w3c/webauthn/wiki/Explainer:-JSON-Serialization-Methods). The built-in methods are compatible with `@github/webauthn-json` encoding, so you can use them as a drop-in substitute. We strongly recommend doing so, since:
```